src/constants.js

```diff
@@ -170,10 +170,12 @@ export const COMPLIANT_SYSTEMS_FILTER_CONFIGURATION = [
   {
     type: conditionalFilterType.checkbox,
     label: 'Compliance',
-    filterString: (value) => `compliant = ${value}`,
+    filterString: (value) => `${value}`,
     items: [
-      { label: 'Compliant', value: 'true' },
-      { label: 'Non-compliant', value: 'false' },
+      { label: 'Compliant', value: 'compliant = true' },
+      { label: 'Non-compliant', value: 'compliant = false' },
```

Comment:
I think what @LightOfHeaven1994 is describing can be fixed by these changes:
```suggestion
      { label: 'Compliant', value: 'compliant = true AND supported_ssg = true' },
      { label: 'Non-compliant', value: 'compliant = false AND supported_ssg = true' },
```

Reply:
@Fewwy can you try David's suggestion? also PR needs to be rebased as we use new tags gql query

Reply:
@LightOfHeaven1994 he's on PTO, I rebased and adjusted the PR, can you please take a look?

Reply:
@skateman Works great now! thanks for the fix <3